keps/sig-node/5554-in-place-update-pod-resources-alongside-static-cpu-manager-policy/README.md

```diff
@@ -0,0 +1,1210 @@
+
+These may be defined in terms of API maturity, [feature gate] graduations, or as
+something else. The KEP should keep this high-level with a focus on what
+signals will be looked at to determine graduation.
+
+Consider the following in developing the graduation criteria for this enhancement:
+* [Maturity levels (`alpha`,      `beta`,      `stable`)][maturity-levels]
+* [Feature gate][feature gate] lifecycle
+* [Deprecation policy][deprecation-policy]
+
+Clearly define what graduation means by either linking to the [API doc
+definition](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning)
+or by redefining what graduation means.
+
+In general we try to use the same stages (alpha, beta, GA), regardless of how the
+functionality is accessed.
+
+[feature gate]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[maturity-levels]: https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+Below are some examples to consider, in addition to the aforementioned [maturity levels][maturity-levels].
+
+#### Alpha
+
+* Feature implemented behind a feature flag
+* Initial e2e tests completed and enabled
+
+#### Beta
+
+* Gather feedback from developers and surveys
+* Complete features A, B, C
+* Additional tests are in Testgrid and linked in KEP
+
+#### GA
+
+* N examples of real-world usage
+* N installs
+* More rigorous forms of testing—e.g., downgrade tests and scalability tests
+* Allowing time for feedback
+
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports, 
+in back-to-back releases.
+
+**For non-optional features moving to GA, the graduation criteria must include
+[conformance tests].**
+
+[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
+
+#### Deprecation
+
+* Announce deprecation and support policy of the existing flag
+* Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
+* Address feedback on usage/changed behavior, provided on GitHub issues
+* Deprecate the flag
+-->
+
+#### Alpha
+
+* Feature implemented behind `InPlacePodVerticalScalingExclusiveCPUs` feature flag.
+* Unit tests and E2E tests covering basic functionality are added.
+* E2E tests covering multiple containers are added.
+* Initial tests are in Testgrid and linked with feature flag `InPlacePodVerticalScalingExclusiveCPUs`.
+* Negative tests are identified and added.
+
+#### Beta
+
+* Gather feedback from developers and surveys.
+* Additional tests are in Testgrid and linked with feature flag `InPlacePodVerticalScalingExclusiveCPUs`.
+* User feedback (ideally from at least two distinct users) is green,
+* 2 examples of real-world usage
+* 2 installs
```

Comment:
You may be setting the bar for beta too high. It is generally pretty hard to gather substantial user feedback from an alpha feature; usually we put these kinds of things under GA requirements.